src/bp-integrations/learndash/library/SyncGenerator.php

```diff
@@ -859,7 +859,7 @@ public function updateBuddypressGroup( $ld_group_id, $groupId ) {
 					'group_id'    => $groupId,
 					'creator_id'  => $ldGroup->post_author,
 					'name'        => $ldGroup->post_title ?: "For Social Group: {$this->ldGroupId}",
```

Comment:
@mahedihasannoman can you make this **For Social Group** string translatable please?